### PR TITLE
Changing from helm to ksonnet install

### PR DESCRIPTION
**Pull request by johnugeorge**
Since ksonnet is recommended way to install, test scripts should use ksonnet way of installation of pytorch operator

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pytorch-operator/44)

### CODE REVIEW

File: scripts/run-tests.sh [hunk 1]
```diff
@@ -29,31 +29,38 @@ NAMESPACE="${DEPLOY_NAMESPACE}"
 REGISTRY="${GCP_REGISTRY}"
 VERSION=$(git describe --tags --always --dirty)
 GO_DIR=${GOPATH}/src/github.com/${REPO_OWNER}/${REPO_NAME}
+APP_NAME=test-app
+KUBEFLOW_VERSION=master
```
Comment (jose5918) [inline scripts/run-tests.sh]: Do you want kubeflow repo changes to potentially affect the operator tests? I'm ok with it but I usually like to have that seperation of concerns

Comment (johnugeorge) [reply scripts/run-tests.sh]: I see your point. I will update KUBEFLOW_VERSION to 0.2 when it is released. Does it sound ok?

**Pull request merged**